Lib/test/audit-tests.py

```diff
@@ -343,7 +343,7 @@ def hook(event, args):
     try:
         # Don't care if this fails, we just want the audit message
         sock.bind(('127.0.0.1', 8080))
-    except error:
+    except Exception:
```

Comment:
You may be more specific and only catch OSError.

Reply:
It's only a test, and I'd rather not risk breaking buildbots over it.